content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -69,21 +59,23 @@ to manage the lifecycle of Istio.
     $ helm init --service-account tiller
     {{< /text >}}
 
-1. Install Istio:
+1. Install the `istio-init` chart to bootstrap all the Istio's CRDs:
+
+    {{< text bash >}}
+    $ helm install install/kubernetes/helm/istio-init --name istio-init --namespace istio-system
+    {{< /text >}}
+
+1. Verify all the Istio's CRDs have been committed in the Kube api-server by checking all the CRD creation jobs complete with success:
+
+    {{< text bash >}}
+    $ kubectl get job --namespace istio-system | grep istio-crd
```

Comment:
Are you sure this technique works?  I haven't tried it, however, the containers can exit and be in a 0/1 COMPLETED state, and still the CRDs lag in being commited in my experience.

Reply:
How about we check the crds number from API-server?
```
kubectl get crds | grep 'istio.io\|certmanager.k8s.io' | wc -l 
```
The result is supposed to be `57`(`51` for `v1.0`, `3` for `v1.1`, `3` for `cert-manager`) for istio v1.1